client/src/pages/CreateAccount/CreateAccount.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import OktaAuth from '@okta/okta-auth-js';
+import { withAuth } from '@okta/okta-react';
+
+import "./CreateAccount.css"
+
+export default withAuth(class CreateAccount extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: '',
+      sessionToken: null
+    };
+    this.oktaAuth = new OktaAuth({ url: process.env.REACT_APP_ISSUER.split('/oauth2')[0] });
+    this.checkAuthentication = this.checkAuthentication.bind(this);
+    this.checkAuthentication();
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleFirstNameChange = this.handleFirstNameChange.bind(this);
+    this.handleLastNameChange = this.handleLastNameChange.bind(this);
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+  }
+
+  async checkAuthentication() {
+    const sessionToken = await this.props.auth.getIdToken();
+    if (sessionToken) {
+      this.setState({ sessionToken });
+    }
+  }
+
+  componentDidUpdate() {
+    this.checkAuthentication();
+  }
+
+  handleFirstNameChange(e){
+    this.setState({firstName:e.target.value});
+  }
+  handleLastNameChange(e) {
+    this.setState({ lastName: e.target.value });
+  }
+  handleEmailChange(e) {
+    this.setState({ email: e.target.value });
+  }
+  handlePasswordChange(e) {
+    this.setState({ password: e.target.value });
+  }
+
+  handleSubmit(e){
+    e.preventDefault();
+
+    console.log("button clicked")
+
+    fetch('/api/users', {
```

Comment:
Is this an Okta route or ours? Can we create a function to reference the utils/API file?